text/line_commands.talon

```diff
@@ -1,4 +1,5 @@
 tag: user.line_commands
+tag: user.navigation_commands
```

Comment:
We don't want to make all of line_commands depend on navigation_commands, since many commands in it work even without navigation. Could you split the commands that require navigation out into a separate file?

Reply:
I disagree, All commands inside this file are navigating somewhere. It's only that the "go line" Method is implemented in Talon by default. In my opinion we should group functionality in files and not let what it depends on determined grouping. 

With that reasoning I actually think this whole file should not exist, but that's another story.

Reply:
I figured we don't need the module and tag at all, soo removed it. Looks cleaner now.